walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -91,28 +137,38 @@
     }
 
     // time range objects are ordered, so you can compare them by order
-    for (int i=0; i<=eventsAsTimeRanges.size()-2; i++) {
+    for (int i = 0; i <= eventsAsTimeRanges.size()-2; i++) {
```

Comment:
should be` i <= eventsAsTimeRanges.size() - 2` (need the spaces around the minus sign)